tests/shared_test_functions.sh

```diff
@@ -42,6 +42,9 @@ out_valgrind="tests-valgrind"
 valgrind_suppressions="${out_valgrind}/suppressions"
 valgrind_suppressions_log="${out_valgrind}/suppressions.pre"
 
+# Print output about test failures.
+VERBOSE=${VERBOSE:-0}
```

Comment:
Just checking... setting VERBOSE to 0 if not defined sounds like it's *not* printing things?  Maybe this just needs the comment clarified.  (e.g., maybe it means "print output about test failures but not test successes" or something like that.)

Reply:
err, yeah.  `VERBOSE=0` means that it only prints `PASS` or `FAIL`.  I'll update the comment.